frontend/src/components/RFI/RFILanding.vue

```diff
@@ -852,152 +799,63 @@
               <v-divider />
 
               <v-row v-for="(obj, index) in model.expansionList" :key="index">
-                <v-col class="col-md-1 col-12 mx-0">
-                  <v-icon
-                    :disabled="isReadOnly"
-                    size="large"
-                    color="blue-darken-4"
-                    class="mt-md-4"
-                    @click="removeObjFromList(index, model.expansionList)"
-                  >
-                    mdi-close
-                  </v-icon>
-                </v-col>
+                <v-icon
+                  :disabled="isReadOnly"
+                  size="large"
+                  color="blue-darken-4"
+                  class="mt-md-6 mx-md-6"
+                  @click="removeObjFromList(index, model.expansionList)"
+                >
+                  mdi-close
+                </v-icon>
+
                 <v-col class="col-md-1 col-12 ml-md-n8">
-                  <v-menu
-                    ref="menufrom"
-                    v-model="obj.menufrom"
-                    v-model:return-value="timefrom"
-                    :close-on-content-click="false"
-                    :nudge-right="40"
-                    transition="scale-transition"
-                    offset-y
-                    max-width="290px"
-                    min-width="290px"
-                  >
-                    <template #activator="{ props }">
-                      <v-text-field
-                        v-model="obj.timefrom"
-                        :disabled="isReadOnly"
-                        label="From"
-                        readonly
-                        v-bind="props"
-                      />
-                    </template>
-                    <v-time-picker
-                      v-if="obj.menufrom"
-                      v-model="obj.timefrom"
-                      :disabled="isReadOnly"
-                      full-width
-                      @click:minute="$refs.menufrom[index].save(timefrom)"
-                    />
-                  </v-menu>
-                </v-col>
+                  <AppTimeInput
+                    v-model="obj.timefrom"
+                    :disabled="isReadOnly"
+                    full-width
+                    max-width="200px"
+                    label="open"
```

Comment:
I think the original label was 'From'. Do we have a requirement for changing it to 'open'?

Reply:
done